QuickOSM/quick_osm.py

```diff
@@ -159,8 +167,40 @@ def open_dialog(self):
         """Create and open the main dialog."""
         from QuickOSM.ui.dialog import Dialog
         dialog = Dialog()
+        self.open_copyright_message(dialog)
         dialog.exec_()
 
+    @staticmethod
+    def open_copyright_message(dialog: QDialog):
+
+        def read_copyright():
+            open_webpage('https://www.openstreetmap.org/copyright')
+            set_setting("copyright_dialog", "OpenStreetMap")
+
+        def know_copyright():
+            set_setting("copyright_dialog", "OpenStreetMap")
+
+        if not get_setting("copyright_dialog"):
+
+            message = QMessageBox(dialog)
+            text = tr(
+                'OpenStreetMap® is open data, licensed under the'
+                ' Open Data Commons Open Database License (ODbL) '
+                'by the OpenStreetMap Foundation.\n'
```

Comment:
Can you split each string when there is a `\n` ? So it doesn't go in the translation process.